templates/base/packages/nextjs/next.config.ts.template.mjs

```diff
@@ -30,9 +31,14 @@ if (isIpfs) {
   };
 }
 
-module.exports = nextConfig;`
+${postConfigContent[0] || ''}
+
+module.exports = ${finalNextConfigName[0]};`
+}
```

Comment:
This will allow to do thing like https://github.com/scaffold-eth/create-eth-extensions/pull/40/files#r1866239279